core/src/main/java/com/datastax/oss/driver/internal/core/metadata/schema/DefaultAggregateMetadata.java

```diff
@@ -149,4 +149,15 @@ public int hashCode() {
         stateFuncSignature,
         stateType);
   }
+
+  @Override
+  public String toString() {
+    return "DefaultAggregateMetadata@"
+        + Integer.toHexString(hashCode())
```

Comment:
Do we want the actual hash code, or `java.lang.System.identityHashCode()`? With the former two semantically equivalent objects would have the same `toString` representation, with the latter they wouldn't.

Reply:
I think semantically equivalent makes the most sense, that way you can tell if two different parser executions produced different results (they always produce distinct instances).